### DISCUSSION TITLE
link only

### DISCUSSION THREAD

**Discussion by teamtacho** [liedekef/events-made-easy#253] [General]
I'd like to set a link to the event directly in the` <li> `line of the event-list
Is the a function to add the link without showing any text. Clicking somewhere in the line the event page will show up.

**Comment by liedekef** [comment]
This is not a question I understand. I'm moving this to the discussion section.
Also: in the event list, the format of events being shown can be changed 100% to your liking, be it a link to the event, a text, an image, ...

**Comment by teamtacho** [comment]
I have several li element in the list, like date, event-name (with the link to the eventpage), author ...
I have set a borderline and a background-color via css to highlight the line. Now I'd like to link to event whereever I click into the line not only on the event-name element.
